src/domain/timeline/calendar/CalendarEventsContainer.tsx

```
@@ -66,10 +66,10 @@ export interface CalendarEventsEntities {
   };
 }
 
-export const CalendarEventsContainer = ({ journeyId, parentSpaceId, children }: CalendarEventsContainerProps) => {
+export const CalendarEventsContainer = ({ spaceId, parentSpaceId, children }: CalendarEventsContainerProps) => {
   const { data: spaceData, loading } = useSpaceCalendarEventsQuery({
-    variables: { spaceId: journeyId!, includeSubspace: !Boolean(parentSpaceId) },
-    skip: !journeyId,
+    variables: { spaceId: spaceId!, includeSubspace: !Boolean(parentSpaceId) },
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)